src/kitsController.ts

```diff
@@ -274,6 +274,11 @@ export class KitsController {
             return false;
         }
 
+        if (this.cmakeTools.workspaceContext.config.skipSelectKit) {
```

Comment:
Unfortunately, we can't do this here because it means the user can never select a kit even if they wanted to later.

I was thinking about this scenario more, and it's possible that you could achieve the same thing by using `"configureOnOpen": false` and updating some code in `async postWorkspaceOpen(info: CMakeToolsFolder)` in `extension.ts`.